SM/migrations/000006_create_links_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE Shift_workers(
+    shiftId BIGINT NOT NULL REFERENCES Shifts(id),
+    userId BIGINT NOT NULL REFERENCES Users(id),
+    PRIMARY KEY (shiftId, userId)
+);
+
+CREATE TABLE Shift_tasks(
```

Comment:
i can recommend to name tables to many:many relations like {first_table_name}_{second table_name}. In your case - shifts_users. 

Also check does first upper letter make sense